front/src/applications/stdcm/utils/formatStdcmConf.ts

```diff
@@ -165,8 +165,8 @@ export const checkStdcmConf = (
       if (arrivalType === 'preciseTime' && arrival) {
         timingData = {
           arrival_time: arrival.toISOString(),
-          arrival_time_tolerance_before: sec2ms(tolerances?.before ?? 0),
-          arrival_time_tolerance_after: sec2ms(tolerances?.after ?? 0),
+          arrival_time_tolerance_before: tolerances?.before?.ms ?? 0,
+          arrival_time_tolerance_after: tolerances?.after?.ms ?? 0,
```

Comment:
Not part of your PR but here `tolerances` (and `before` / `after`) are always defined, we don't need the `?` (we just put it in the origin or destination path item depending of which one has `arrivalType === 'preciseTime'`)
```suggestion
          arrival_time_tolerance_before: tolerances.before.ms,
          arrival_time_tolerance_after: tolerances.after.ms,
```

Reply:
Added a commit to clean this up!